src/app/collection/[collectionId]/collection.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { CollectionCarousel } from "@/components/CollectionCarousel";
+import { PageLayout } from "@/components/layout";
+import { useCollectionContext } from "@/contexts";
+import { PreviewTitleDto } from "@/dtos";
+import { useEffect, useState } from "react";
+import { styled } from "styled-components";
+
+const CarouselContainer = styled.div`
+	width: 100vw;
+	margin-left: calc(-50vw + 50%);
+`;
```

Comment:
```suggestion
// This container is needed to extend the width to the edges of the screen, outside the PageLayout.
// TODO: Update page layout to use grid.
const CarouselContainer = styled.div` 
	width: 100vw;
	margin-left: calc(-50vw + 50%);
`;
```
I'll make another PR at some point, which should make this unnecessary.